scripts/vim-patch.sh

```diff
@@ -26,7 +26,28 @@ usage() {
   echo "The default is '${VIM_SOURCE_DIR_DEFAULT}'."
 }
 
+# Checks if a program is in the user's PATH, and is executable.
+check_executable() {
+  if [[ ! -x $(command -v "${1}") ]]; then
```

Comment:
We try to avoid bashisms in our scripts, even if we end up needing bash for some things.